src/grid/becke.py

```diff
@@ -22,28 +22,36 @@
 
 import warnings
 
+from grid.utils import get_cov_radii
+
 import numpy as np
 
 
 class BeckeWeights:
     """Becke weights functions holder class."""
 
-    def __init__(self, radii, order=3):
+    def __init__(self, radii=None, order=3):
         r"""Initialize class.
 
         Parameters
         ----------
-        atom_nums : np.ndarray(M,)
-            Atomic number of :math:`M` atoms in molecule.
-        radii : np.ndarray(M,), optional
-            Atomic radii of :math:`M` atoms in molecule.
+        radii : dict, optional
+            Dictionary of atomic number and corresponding atomic radius.
+            If None, Bragg-Slater empirically measured covalent radii are used.
         order : int, optional
             Order of iteration for switching function.
 
         """
         if not isinstance(order, int):
             raise ValueError(f"order should be an integer, got {type(order)}")
-
+        if radii is not None:
+            if not isinstance(radii, dict):
+                raise TypeError(f"radii should be a dictionary, got {type(radii)}")
+            if not np.all([isinstance(k, int) for k in radii.keys()]):
+                raise TypeError(f"radii keys should be integers.")
```

Comment:
If an extra dict for radii is provided, it may update the default radii dict. So the user don't have to offer every atomic radii.

Reply:
They don't have to provide a radius for every atomic number, just the atomic number they would use later when they call `BeckeWeight` instance is enough. But, if `radii=None`, by default all atomic radii are loaded.

What do you mean by updating the dictionary? Are you talking about new functionality?

Reply:
I assume i have a molecule with 10 different atoms, but I only want to customize one's radius while all the others with default value. In this case, I have to go check the default radii for all the other atoms and put them in the dict.
What I mean is it generate a dict for all atoms, and update the value from arg radii if the user intend to customize.

Reply:
That's a very good point @tczorro, I added a small patch to include this feature. Thanks.